test/app/model/ChessEngine.spec.ts

```diff
@@ -142,4 +143,118 @@ describe('Chess Engine', () => {
       ),
     ).toThrowError('There is no piece on this square.');
   });
+
+  describe('If player wants to move piece that check if it causes checking of his king.', () => {
```

Comment:
Proponuję:
Check if player's possible move will cause checking of his king

Reply:
Dzięki za ten komentarz. Przyda się